### PR TITLE
Horizon

### PR DESCRIPTION
**Pull request by theMetalkate**
Create Horizont component:

Screenshot 2020-03-22 at 11 38 10
Screenshot 2020-03-22 at 11 38 32

### CODE REVIEW

File: src/horizon.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    text: string
+    url: string
+}
+
+export default function Horizon(entries: HorizonProp[]) {
+    return (
+        <Wrapper>
+            {entries.map((horizonEntry) => {
+                return (
+                    <Item href={horizonEntry.url}>
```
Comment (Entkenntnis) [inline src/horizon.tsx]: please add a `key` prop here

File: src/horizon.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            })}
+        </Wrapper>
+    )
+}
+
+const Wrapper = styled.div`
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+    padding: 2rem 1.5rem 1.5rem;
```
Comment (Entkenntnis) [inline src/horizon.tsx]: use px here

File: src/horizon.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    max-width: 400px;
+    width: 98%;
+    @media (max-width: ${props => props.theme.breakpoints.sm}) {
+        max-width: 400px;
+    }
+`
+
+const Headline = styled.p`
+    font-weight: bold;
+    font-size: 20px;
```
Comment (Entkenntnis) [inline src/horizon.tsx]: use `rem` for font size

File: src/horizon.tsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
+import * as React from 'react'
```
Comment (Entkenntnis) [inline src/horizon.tsx]: for consistency, change to `import React from 'react'`

File: src/horizon.tsx (hunk 1 above)
Comment (theMetalkate) [reply src/horizon.tsx]: please take a look

File: src/horizon.tsx (hunk 2 above)
Comment (theMetalkate) [reply src/horizon.tsx]: fixed

File: src/horizon.tsx (hunk 4 above)
Comment (theMetalkate) [reply src/horizon.tsx]: fixed

File: src/horizon.tsx (hunk 1 above)
Comment (Entkenntnis) [reply src/horizon.tsx]: key={index} is probably enough

**Pull request merged**